cmd/root.go

```diff
@@ -117,17 +119,46 @@ func loadBranches(ctx context.Context, remote shared.Remote, defaultBranchName s
 	prs := []shared.PullRequest{}
 	orgs := shared.GetQueryOrgs(repoNames)
 	repos := shared.GetQueryRepos(repoNames)
-	for _, queryHashes := range shared.GetQueryHashes(branches) {
-		json, err := connection.GetPullRequests(ctx, remote.Hostname, orgs, repos, queryHashes)
-		if err != nil {
-			return nil, err
-		}
 
-		pr, err := toPullRequests(json)
-		if err != nil {
-			return nil, err
+	type pullRequestResult struct {
+		prs []shared.PullRequest
+		err error
+	}
+
+	queryHashes := shared.GetQueryHashes(branches)
+	results := make(chan pullRequestResult, len(queryHashes))
```

Comment:
I want to unify the naming with the goroutine below, so how about calling it something like `prChan`?

Reply:
Renamed it to `prChan`!